frontend/views/components/Tooltip.vue

```diff
@@ -131,7 +131,7 @@ export default {
       this.styles = {
         transform: `translate(${x}px, ${y}px)`,
         pointerEvents: this.manual ? 'initial' : 'none',
-        backgroundColor: this.manual ? 'transparent' : 'var(--text_0)',
+        backgroundColor: this.manual ? 'transparent' : 'var(--background_0)',
```

Comment:
Looks like this change has created a bug where the tooltip doesn't properly show.
![image](https://user-images.githubusercontent.com/17641213/100310825-ff8e6a00-3012-11eb-85c4-bf10bb4b1bed.png)